src/Menu/Menu.Component.js

```diff
@@ -82,11 +105,7 @@ export const MenuComponent = () => {
                         name: 'addonProps',
                         description: 'object - additional props to be spread to the addon section'
                     },
-                    { name: 'link', description: 'string - a router link. Use either \'url\' or \'link\'' },
-                    {
-                        name: 'linkProps',
-                        description: 'object - additional props to be spread to the Menu Item links'
-                    },
+                    { name: 'children', description: 'component - can be used to pass React Router <Link> or any other component which emits an <a>.' },
```

Comment:
Be careful if https://github.com/SAP/fundamental-react/pull/291 gets merged first, you'll need to move the description into the new object.

Reply:
It's been merged so you can be the first new consumer of the updated prop documentation. 👍

Reply:
Fixed!